functions/Get-DbaLastBackup.ps1

```diff
@@ -0,0 +1,158 @@
+﻿Function Get-DbaLastBackup
+{
+<#
+.SYNOPSIS
+Get date/time for last known backups
+
+.DESCRIPTION
+Retrieves and compares the date/time for the last known backups, as well as the creation date/time for the database.
+
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Databases
+Return information for only specific databases
+
+.PARAMETER Exclude
+Return information for all but these specific databases
+
+.PARAMETER Detailed
+Shows detailed information
+
+.NOTES
```

Comment:
.NOTES
Author: Klaas Vandenberghe ( @PowerDBAKlaas )
	
dbatools PowerShell module (https://dbatools.io)
Copyright (C) 2016 Chrissy LeMaire
This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.